automated-testing/templates/test_type_template.md

```diff
@@ -1,27 +1,58 @@
-# ~Test type~
+# Insert Test Technique Name Here
 
-The document should start with a brief overview about the test type and what is covered in this document, the goal here is to provide a high-level description to help the reader understand what is covered to decide whether to continue reading or not.
+Put a 2-3 sentence overview about the test technique here.
 
-## Why ~Test type~ [The Why]
+## When To Use
 
-Start by describing the problem that this test type addresses, this should focus on the motivation behind the test type to aid the reader correlate this test type to the problem they are trying to resolve.
+### Problem Addressed
 
-## ~Test type~ Design Blocks [The What]
+Describing the problem that this test type addresses, this should focus on the motivation behind the test type/technique to help the reader correlate this technique to their problem.
 
-In this section, describe the test type, its components, and how they interact to solve the problem described above.
+### When to Avoid
 
-## Applying the ~test type~ [the how]
+Describe when NOT to use, if applicable.
 
-In this section, describe what good testing looks like for this test type, discuss some of the best practices, discuss pitfalls to avoid, and finally discuss some of the common tools used to apply the test type, if any.
+### ROI Tipping Point
 
-## ~Test type~ Frameworks and Tools
+How much is enough?  For example, some opine that unit test ROI drops significantly at 80% block coverage and when the codebase is well-exercised by real traffic in production.
 
-In this section, describe various test frameworks and tools, their pros and cons, and provide with the links to where to get more information.
+### Applicable to
 
-## Conclusion
+- [ ] Dev Desktop
```

Comment:
I'm not sure Dev Desktop is the best term here. Maybe something like local deployment or development instead?

Also, what if I'm using a notebook ;)

Reply:
Local Developer Experience perhaps?